pvcollada20_schema/pvcollada_schema_0.1.xsd

```diff
@@ -543,23 +546,24 @@
   <!-- Inputs container for instance_combiner_Dc; each input must have exactly one instance child (DC combiner or string) -->
   <xs:complexType name="combiner_dc_inputs_type">
     <xs:sequence>
+      <!-- Do we need this input? -->
       <xs:element name="input" type="combiner_dc_input_type" minOccurs="1" maxOccurs="unbounded"/>
     </xs:sequence>
   </xs:complexType>
   
-  <!-- Input for instance_combiner_dc/inputs elements; it must have exactly one child chosen from the allowed types -->
+  <!-- Input for instance_combiner_dc/inputs elements; it must have one or more strings or one or more combiner boxes -->
   <xs:complexType name="combiner_dc_input_type">
-    <xs:choice minOccurs="1" maxOccurs="unbounded">
```

Comment:
The `maxOccurs="unbounded"` here isn't good.
I think the author intended to place it on the  `instance_combiner_dc` and `string` elements, so I did that.